src/character.cpp

```diff
@@ -2645,3 +2647,59 @@ float Character::healing_rate( float at_rest_quality ) const
 
     return final_rate;
 }
+
+float Character::healing_rate_medicine( float at_rest_quality, const body_part bp ) const
+{
+    float rate_medicine = 0.0f;
+    float bandaged_rate = 0.0f;
+    float disinfected_rate = 0.0f;
+    float awake_rate = mutation_value( "healing_awake" );
```

Comment:
Variable is not used later in the method, so it should either be removed or used.